content/info/call-participation/doctoral-colloquium.md

```diff
@@ -0,0 +1,44 @@
+---
+title: doctoral-colloquium
+layout: page
+permalink: /info/call-participation/doctoral-colloquium
+contact: [committee-name]@ieeevis.org
```

Comment:
Please fix the contact email. For the rest it looks good

Reply:
Sorry, should be fixed now.